samples/layouts/expansion-panel/properties-and-events/src/ExpansionPanelPropertiesAndEvents.css

```diff
@@ -4,10 +4,14 @@
     height: 300px;
 }
 
+#exp-panel{
+    width: 100%;
+}
```

Comment:
This shouldn't be needed. You can simply target the expansion panel by tag selector `igc-expansion-panel`. Also, I am not sure why setting the width to a 100% is explicitly needed.

Reply:
When the width of the expansion panel is not set, the header size is being changed when the panel is expanded/collapsed